fuzzer/svc_enc_fuzzer.cpp

```diff
@@ -545,7 +545,7 @@ bool Codec::initEncoder(const UWORD8 *data)
     isvce_init_ip_t sInitIp{};
     isvce_init_op_t sInitOp{};
 
-    std::vector<UWORD32> sMaxBitrates(mNumSpatialLayers, 240000000);
+    std::vector<UWORD32> sMaxBitrates(mNumSpatialLayers, 6000000);
```

Comment:
is this change really necessary? does this mean max bitrate for each layer is 6mbps? isn't that low?

Reply:
This is not needed. I will revert it.